src/ACEtk/block/details/BaseAngularDistributionBlock/src/insertZero.hpp

```diff
@@ -0,0 +1,6 @@
+std::vector< DistributionData >
+insertZero( DistributionData&& zero, std::vector< DistributionData >&& data ) {
```

Comment:
This function can be made static.

Reply:
This has been corrected already in another PR